src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('new-user');
       return;
     }
     if (isUndefinedUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('undefined-user');
       return;
     }
     if (isSecurityImage) {
+      // TODO: Newer versions of qtip will remove aria-describedby on their own when destroy() is called.
+      el.removeAttr('aria-describedby');
```

Comment:
qtip sets the aria-describedby property when it's initialized. I don't know whether that property is working correctly or not. qtip.destroy() gets called before this function, but it leaves behind the (now-erroneous) aria-describedby it already set.